lib/propmangle.js

```diff
@@ -150,6 +150,11 @@ function mangle_properties(ast, options) {
             if (!(keep_quoted && node.quote))
                 node.key = mangle(node.key);
         }
+        else if (node instanceof AST_ConciseMethod && node.name && node.key.name) {
```

Comment:
Hmm... I thought `Unquoted` in the test would exercise this. Now I'm wondering whether this code is used at all.